### PR TITLE
created the popup array and window

### PR DESCRIPTION
**Pull request by alexmedinasf**
Index: Created the popup structure
Css: styled images and the popup page
Js: Created functions and array for the popup

### CODE REVIEW

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Alexandre, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project, but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- Modal working as intended
- ReadMe looks good


## Required Changes ♻️
_Kindly check the comments under the review._


<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._,

File: css/style.css [hunk 1]
```diff
@@ -522,3 +524,116 @@ form {
...
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  max-width: 100%;
+  height: 95vh;
+  padding: 10px;
+  margin-top: 100px;
+  margin-bottom: 100px;
+}
+
+.modal-container {
```
[the remark this answers is not in the document]
Comment (alexmedinasf) [reply css/style.css:538]: @rloterh you checked the wrong template.

File: js/index_dom.js [hunk 2]
```diff
@@ -19,4 +20,146 @@ navLink.forEach((element) => element.addEventListener('click', () => {
...
+
+      'HTML',
+      'Bootsrap',
+      'Ruby',
+
+    ],
+    liveLink: 'https://alexmedinasf.github.io/Porfolio/',
+    sourceLink: 'https://github.com/alexmedinasf/Porfolio',
+    description: "Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+  },
```
Comment (roshan-bajgain) [inline js/index_dom.js:53]: - [ ] [Optional] I would suggest you please implement the logo on `See live` and `See source` buttons. See image below for reference
- [ ] [Optional] Kindly remove text-decoration as well.
### Figma design
figma
### Your design
your

File: css/style.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (rloterh) [reply css/style.css:538]: @alexmedinasf 
Kindly let me confirm once more and get back to you!

Cheers!

File: js/index_dom.js [hunk 3]
```diff
@@ -19,4 +20,146 @@ navLink.forEach((element) => element.addEventListener('click', () => {
...
+  projectModal.style.display = 'block';
+}
+
+projectButtons.forEach((projectButton) => {
+  const { id } = projectButton;
+  projectButton.addEventListener('click', () => openProjectModal(id));
+});
+closeBtn.addEventListener('click', () => {
+  projectModal.style.display = 'none';
+});
```
Comment (roshan-bajgain) [inline js/index_dom.js:165]: - [ ] I would suggest you please add the `X` button on the top of the page as in the Figma design.
- [ ] I am not able to scroll your pop-up widow for the mobile version as it is not showing the whole content vertically please fix it.
### Your design
sssssssssss

**Review by roshan-bajgain — APPROVED** [review]
Hi @alexmedinasf, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions
- [ ] [Optional] Please make change to the popup window of mobile design.
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (rloterh) [reply css/style.css:538]: I carefully cloned your project once more and confirming it in a private mode of a browser, and the result is the same. However, I apologize I that missed step of checking the implementation of the see live and see source buttons, as indicated in the subsequent review. Kindly note that this was not a deliberate attempt.

Cheers, and happy coding! 🤝😃

**Review by nicupop729 — COMMENTED** [review]
Hello @alexmedinasf 👋,

_This is an additional review as a result of the second opinion request._

I noticed that the project was approved and you didn't wait for the Second Opinion Review you asked for. 
In the future, if you will find the need to ask again for a Second Opinion Review, I kindly ask you to wait for a response before asking for another review in your dashboard.

Let me mention the following:
- the project should have been invalidated by @rloterh because in a pair programming activity you worked alone;
- both @rloterh and @roshan-bajgain missed mentioning that you were supposed to refactor the HTML file; the work cards hardcoded in the HTML need now to be rendered dynamically with the use of JS and the hardcoded ones need to be deleted.

image

Please remember that the purpose of this exercise is for you to practice DOM manipulation and get you prepared for a job interview which could lead to a real job. 
I strongly recommend that you implement the above requirements you missed.

Keep rocking 🚀!

**Pull request merged**